docs/daily/2024-12-21.md

```
@@ -0,0 +1,109 @@
+  - Wizard파일 getter 체크해보기
+
+
+- `late init var`랑 `!!` 쓰지 마라. `!!`는 nullable인거 쓸때 내가 null이 아님을 보증하는거임.
+```kotlin
+class Wand(name: String?) {
+
+var name = require(name?.length ?: 0 > 3) // 앨비스 연산자로 앞에가 널이면 기본값 주는 문법도 있음
+}
+
+fun main() {
+	vall wand = Wand(null)
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)